en/sql/schema/stored_routine_stmt.rst

```diff
@@ -0,0 +1,366 @@
+    
+    SELECT * FROM db_stored_procedure_args WHERE sp_name = 'athlete_add';
+    
+::
+    
+    sp_name   index_of  arg_name  data_type      mode
+    =================================================
+     'athlete_add'                   0  'name'                'STRING'              'IN'
+     'athlete_add'                   1  'gender'              'STRING'              'IN'
+     'athlete_add'                   2  'nation_code'         'STRING'              'IN'
+     'athlete_add'                   3  'event'               'STRING'              'IN'
+
+
+DROP PROCEDURE
+==============
+
+In CUBRID, A stored proceudre can be deleted using the **DROP PROCEDURE** statement.
+Also, you can delete multiple stored procedures at a time with several *procedure_name*s separated by a comma (,).
+
+::
+
+    DROP PROCEDURE procedure_name [{ , procedure_name , ... }]
+
+*   *procedure_name*: Specifies the name of procedure to delete
+
+.. code-block:: sql
+
+    DROP PROCEDURE hello, sp_int;
+
+A stored procedure can be deleted only by the user who published it or by DBA members. 
+For example, if a **PUBLIC** user published the 'sp_int' stored procedure, only the **PUBLIC** or **DBA** members can delete it.
+
+.. _create-function:
+
+CREATE FUNCTION
+=================
+
+Create stored function using the **CREATE FUNCTION** statement.
+The other languages except Java do not support stored function. 
+In CUBRID, only Java can implement stored function.
+See details of how to use Stored Function, please refer to the :doc:`/sql/jsp`.
+
+::
+
+    CREATE [OR REPLACE] FUNCTION function_name [(<parameter_definition> [, <parameter_definition>] ...)] RETURN sql_type
+    {IS | AS} LANGUAGE JAVA <java_call_specification>
+    COMMENT 'sp_comment_string';
+    
+        <parameter_definition> ::= parameter_name [IN|OUT|IN OUT|INOUT] sql_type [COMMENT 'param_comment_string']
+        <java_call_specification> ::= NAME 'java_method_name (java_type [,java_type]...) [return java_type]'
+
+*   *function_name*: Specifies the name of the stored function(maximum 254 bytes).
+*   *parameter_name*: Specifies the name of the parameter (maximum 254 bytes).
+*   *sql_type*: Specifies the data type of the parameter or of the return value. See details on the data types that can be used for the parameter, refer to the :ref:`jsp-type-mapping`.
+*   *param_comment_string*: Specifies comment of the parameter.
+*   *sp_comment_string*: Specifies comment of the stored function.
+*   *java_method_name*: Specifies the name of Java method name, including the name of the class it belongs to.
+*   *java_type*: Specifies the Java data type. See details on the Java data types that can be used to return, refer to the :ref:`jsp-type-mapping`.
+
+You must publish Java classes by using Java Call Specifications (<*java_call_specification*>) becuase it is not known how a function in a class will be called by SQL statements or Java applications when Java classes are loaded.
```

Comment:
```suggestion
You must publish Java classes by using Java Call Specifications (<*java_call_specification*>) because it is unknown how a function in a class will be called by SQL statements or Java applications when Java classes are loaded.
```